app/services/profiles/import_prisoner_category.rb

```diff
@@ -0,0 +1,12 @@
+module Profiles
+  class ImportPrisonerCategory
+    def initialize(profile)
+      @profile = profile
+      @nomis_booking_id = @profile&.person&.latest_nomis_booking_id
+    end
+
+    def call
+      @profile.update(GetPrisonerCategoryAttributes.new(@nomis_booking_id).call) if @nomis_booking_id.present?
```

Comment:
do we need the nomis booking id check here as well as `GetPrisonerCategoryAttributes`? This will avoid updating/clearing the category data if it changes, but if we keep it on `GetPrisonerCategoryAttributes` and remove here we will always update it

Reply:
update this is only for the rake task so is ok